test.js

```diff
@@ -4,6 +4,8 @@ import {createLogUpdate} from './index.js';
 
 const setup = options => {
 	const terminal = new Terminal(options);
+	terminal.rows = options.rows;
+	terminal.columns = options.columns;
```

Comment:
https://github.com/sindresorhus/log-update/blob/73424532e029c483b0f5f7155204e33989884ec5/index.js#L27-L32
This part of the `fitToTerminalHeight` function was never called in tests.

It was because `this.options.rows` and `this.options.columns` was not assigned to `this.rows`/`this.columns`.
https://github.com/Gottox/terminal.js/blob/bff0372cfaf7ad9eedeec2ca4c7a63f6fcdee089/lib/terminal.js#L28-L34